ui/src/components/create-community.tsx

```diff
@@ -22,9 +22,9 @@ export class CreateCommunity extends Component<any, any> {
       <div class="container">
         <div class="row">
           <div class="col-12 col-lg-6 offset-lg-3 mb-4">
-            <h5>
+            <h2>
```

Comment:
Sorry, i don't get your argument, what do you mean by changing it? 

As headings should be ordered from h1 to h6 it made more sense to make them h2 as they are the first level of item headings being used on a page. (You could even go as far as to make them h1).

Also, i couldn't see a visual difference between those headings, i didn't spend time to dive into any of the styles really, but it seems they are not yet defined?

Reply:
Ah I see, this was because of an error in my css which I've now fixed, having to do with resizing the headers in markdown (and I accidentally applied it to others around the site).

Here I care more about the sizing than whether the specific h tag, leave these as h5 pls.

Reply:
I do think that is the wrong way to think about it, headings are not just some HTML elements, they create hierarchy, like a normal document. So if you want to make accessibility a priority, having this hierarchy as much as possible is a good way to start.

I mean, for now it could stay that way if you really want, but it kind of defeats the purpose a little bit to use the element that has the correct styling at that point.

Reply:
Ya that's okay, just leave them as h5. There's no h1s either, and I value bootstrap styling and the preferences of the front end framework over anything else.

Reply:
I reverted it back to h5 :)